packages/get-credentials/test/get-credentials.spec.js

```diff
@@ -0,0 +1,104 @@
+import fs from 'fs'
+import sinon from 'sinon'
+
+import {
+  getCredentials,
+  getCredentialsFromEnvironment,
+  setCredentialsFromEnvFile,
+} from '../src/index'
+import { homepage } from '../package.json'
+
+describe('getCredentials', () => {
+  test('should reject error when missing project key argument', (done) => {
+    getCredentials()
+      .then(done.fail)
+      .catch((errors) => {
+        expect(errors[0].message).toBe('Project Key is needed')
+        done()
+      })
+  })
+
+  test('should reject an array of all errors', (done) => {
+    getCredentials('forgery')
+      .then(done.fail)
+      .catch((errors) => {
+        expect(errors.length).toBe(2)
+        expect(errors[0].message).toBeTruthy()
+        expect(errors[1].message).toBeTruthy()
+        done()
+      })
+  })
+})
+
+describe('getCredentialsFromEnvironment', () => {
+  const sandbox = sinon.sandbox.create()
+
+  afterEach(() => sandbox.restore())
+
+  test('should resolve credentials from environment variables', () => {
+    sandbox.stub(process, 'env', { CT_STROOPWAFEL: 'nyw:les' })
```

Comment:
You don't need `sinon`. You can simply do `process.env.CT_STROOPWAFEL='nyw:les'`, the real env var won't be overridden.

```bash
$ export FOO=bar
$ echo $FOO
bar
$ npm t

 PASS  packages/xxx.spec.js
 ...
Ran all test suites.
$ echo $FOO
bar
```

and the test was

```js
it('should mock env var', () => {
  process.env.FOO = '123'
  expect(process.env.FOO).toEqual('123')
})
```